lib/ansible/galaxy/collection.py

```diff
@@ -320,6 +320,7 @@ def from_name(collection, apis, requirement, force, parent=None):
                 resp = json.load(open_url(n_collection_url, validate_certs=api.validate_certs, headers=headers))
             except urllib_error.HTTPError as err:
                 if err.code == 404:
+                    display.vvvv("Collection '%s' is not available from server '%s'" % (collection, api.name))
```

Comment:
actually, we would prefer 3 v's because

https://github.com/ansible/awx/blob/43d816b6e4c7ea01c31e42e713101cf51011b534/awx/main/tasks.py#L1887